src/main.dart

```diff
@@ -1,8 +1,99 @@
-// TODO: Run task list app
-void main(List<String> args) {
-  print('Hello World');
+import 'dart:developer';
+
+import 'dart:io';
+
+class Task {
+  int? taskID;
+  String? name;
+  String? description;
+  bool? isCompleted;
```

Comment:
this should never be null
```suggestion
  bool isCompleted;
```